Dockerfile

```diff
@@ -16,7 +16,9 @@ LABEL org.label-schema.vcs-ref=$VCS_REF
 LABEL org.label-schema.build-date=$BUILD_DATE
 LABEL org.opencontainers.image.vendor="Saxon State and University Library Dresden"
 LABEL org.opencontainers.image.title="OCR-D Controller"
+LABEL org.opencontainers.image.description="Path to network implementation of OCR-D"
 LABEL org.opencontainers.image.source="https://github.com/slub/ocrd_controller"
+LABEL org.opencontainers.image.documentation="https://github.com/slub/ocrd_controller/blob/${VCS_REF}/README.md"
```

Comment:
Yes. For ocrd_kitodo, we can reference the mkdocs site at this point.

Reply:
Unfortunately this documentation is rather unspecific for the respective module and would leave it here rather with the Readme.

Reply:
Ok, never mind.